secure_message/resources/messages.py

```diff
@@ -89,14 +89,14 @@ def _message_save(message, is_draft, draft_id):
         """Saves the message to the database along with the subsequent status and audit"""
         save = Saver()
         save.save_message(message.data)
+        save.save_msg_actors(message.data.msg_id, message.data.msg_from, message.data.msg_to[0], g.user.is_internal)
```

Comment:
Will `g` always have a `user` attribute? If not, what is the expected behaviour? As of Flask 0.10 `g` implements a `get` method that returns None by default.

Reply:
We always have it. When we read the jwt as part of the authentication we create a userobject and set  g.user  to be this user object. This is done prior to all calls , so we are safe then in reading it anywhere in the api